marklogic-data-hub/src/test/java/com/marklogic/hub/EndToEndFlowTests.java

```diff
@@ -171,120 +177,126 @@ public static void teardown() {
         DataHub dataHub = getDataHub();
         List<DynamicTest> tests = new ArrayList<>();
 
-        allCombos((codeFormat, dataFormat, flowType) -> {
+        allCombos((codeFormat, dataFormat, flowType, useEs) -> {
+            if (useEs) {
+                return;
```

Comment:
we've got this return, but the code below also has `useEs ? "-es" : ""`. Unclear why we'd return here.

Reply:
because we don't want to run ES tests in this case. and the `useEs ? "-es" : ""` was me going copy/paste crazy. I will remove the  `useEs ? "-es" : ""` to make it a bit more clear. maybe a comment too :)